Use normative key words for MeterProvider's Meter API parameters (#3069)
open-telemetry/opentelemetry-specification: specification/metrics/api.md

  implementation which is not even observability-related). A MeterProvider could
  also return a no-op Meter here if application owners configure the SDK to
  suppress telemetry produced by this library.

> This API MAY accept the following parameters:

Author: ```suggestion
Additionally, this API MAY accept the following parameters:
```

> Additionally, this API MAY accept the following parameters:

Reviewer: I understand that the API must handle the parameters if provided. It is only optional for the user to provide them. Right? If so then this is not clear. How about

```suggestion
Additionally, this API MUST allow using the following optional parameters:
```

Author: Updated, PTAL.
